symposion/reviews/views.py

```diff
@@ -93,13 +94,21 @@ def review_section(request, section_slug, assigned=False):
     queryset = ProposalBase.objects.filter(kind__section=section)
 
     if request.method == "POST" and can_manage:
-        pk = request.POST['pk']
-        status = request.POST['status']
-        base_obj = queryset.get(pk=pk)
-        p_type = section_slug.rstrip('s').replace('-', '')
-        proposal = getattr(base_obj, 'pycon%sproposal' % p_type)
-        proposal.overall_status = status
-        proposal.save()
+        pk_string = request.POST['pk']
+        if pk_string != '':
+            pk_list =  [int(i) for i in pk_string.split(',')]
+            for pk in pk_list:
+                status = request.POST['status']
+                base_obj = queryset.get(pk=pk)
+                p_type = section_slug.rstrip('s').replace('-', '')
+                proposal = getattr(base_obj, 'pycon%sproposal' % p_type)
+                proposal.overall_status = status
+                proposal.save()
+        else:
+            messages.add_message(
+                request, messages.ERROR,
+                (u"Please select at least one application"))
```

Comment:
This works fine, but there are also shortcut methods for the various message levels that you can use if you prefer, like: `messages.error(request, _("text")`).